 src/SoundEngine.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/SoundEngine.cpp b/src/SoundEngine.cpp
index 979057d..d3609b5 100644
--- a/src/SoundEngine.cpp
+++ b/src/SoundEngine.cpp
@@ -18,7 +18,7 @@ namespace {
 // Functions and data that control the engine's state
 
 struct SoundSourceData {
-    using data_t = union {
+    union data_t {
         Mix_Chunk* chunk = nullptr;
         Mix_Music* music;
     };
@@ -439,7 +439,7 @@ bool set_volume(Sound sound, float volume) {
     if (source_is_music(data.source)) {
         Mix_VolumeMusic(static_cast<int>(MIX_MAX_VOLUME * volume));
     } else {
-        Mix_Volume(data.channel, static_cast<int>(MIX_MAX_VOLUME * volume));
+        Mix_Volume(-1, static_cast<int>(MIX_MAX_VOLUME * volume));
     }
 
     return true;
